README.md

```diff
@@ -40,6 +40,7 @@ To start a dev server running all the modules simultaneously, run:
 ```bash
 yarn start
 ```
+Note: You need to specify the sources for yarn start to work.For example - `yarn start --sources 'packages/esm-patient-search-app'`
```

Comment:
This should be true when running a specific module but not for `yarn start`. Did you see [this](https://github.com/openmrs/openmrs-esm-patient-management#:~:text=To%20start%20a%20dev%20server%20for%20a%20specific%20module%2C%20run%3A)?

Reply:
Hello @kdaud yes I already saw that. But yarn start will always throw error if we do not specify the sources. Initially I modified the `yarn start` command to `yarn start --sources packages/esm-*-app` but I modified the docs again after the suggestions. Any suggestion on how exactly should we put this in documentation will be a great help. 
Thanks for reviewing.

Reply:
Could you run again the command `yarn start`?

Reply:
I tried but getting [error](https://pastebin.com/mrGRQM16).

Reply:
Yes `yarn start` won't work without --sources
 
Maybe we can have something similar to https://github.com/openmrs/openmrs-esm-patient-chart

Reply:
yes, I guess that will be optimal solution for the issue. Should I go ahead and update the PR and make it similar to this?

Reply:
yes. Go ahead.